install_config/configuring_admission_control.adoc

```diff
@@ -0,0 +1,161 @@
+you are doing. Future versions of the product may use a different set of plugins and may change their ordering. If you do override the
+default list of plugins in the master config, you are responsible for updating it to reflect requirements of newer versions of the
+OpenShift master.
+====
+
+=== Overriding the list of admission control plugins for OpenShift resources
+
+The list of admission control plugins for OpenShift resources can be configured in the
+link:master_node_configuration.html#master-configuration-files[master-config.yaml]
+as follows:
+
+[source,yaml]
+----
+admissionControl:
+  pluginOrderOverride:
+  - OriginNamespaceLifecycle
+  - BuildByStrategy
+  - YourPlugin
+----
+
+If the `pluginOrderOverride` setting is empty or not present, then the default set of plugins will be used.
+
+The default list of admission control plugins for OpenShift resources is:
+
+[source, yaml]
+----
+- OriginNamespaceLifecycle
+- BuildByStrategy
+----
+
+* *OriginNamespaceLifecycle* Enforces that a
+link:../architecture/core_concepts/projects_and_users.html#namespaces[Namespace]
+must exist before creating resources in it. It also prevents creation of resources in a project
+that is in the process of getting deleted.
+
+* *BuildByStrategy* Restricts creation of
+link:../architecture/core_concepts/builds_and_image_streams.html#builds[Builds]
+and
+link:../dev_guide/builds.html#defining-a-buildconfig[BuildConfigs]
+to a strategy that the current user is
+link:../admin_guide/securing_builds.html[allowed to use].
+
+=== Overriding the list of admission control plugins for Kubernetes resources
+
+The list of admission control plugins for Kubernetes resources can be configured in the
+link:master_node_configuration.html#master-configuration-files[master-config.yaml]
+as follows:
+
+[source,yaml]
+----
+kubernetesMasterConfig:
+  admissionConfig:
+    pluginOrderOverride:
+    - NamespaceLifecycle
+    - OriginPodNodeEnvironment
+    - YourPlugin
+    - ...
+----
+
+The default list of admission control plugins for Kubernetes resources is:
+
+[source, yaml]
+----
+- NamespaceLifecycle
+- OriginPodNodeEnvironment
+- LimitRanger
+- ServiceAccount
+- SecurityContextConstraint
+- ResourceQuota
+- SCCExecRestrictions
+----
+
+* *NamespaceLifecycle* Enforces that a
+link:../architecture/core_concepts/projects_and_users.html#namespaces[Namespace]
+must exist before creating resources in it. It also prevents creation of resources in a project
+that is in the process of getting deleted.
+
+* *OriginPodNodeEnvironment* Ensures that a
+link:../architecture/core_concepts/pods_and_services.html#pods[Pod]'s node selector matches at least one node
+in the cluster.
+
+* *LimitRanger* Enforces resource usage
+link:../dev_guide/limits.html[limits] for pods created in a given Namespace.
+
+* *ServiceAccount* Ensures that all pods are associated with a valid
+link:../dev_guide/service_accounts.html[service account] and that the
```

Comment:
because the setup aspects are irrelevant for developers, but crucial for admins. it felt weird to tell admins how to set up serviceaccounts but link to the dev section to read about what they actually are.